src/lib/ngsi/Scope.cpp

```diff
@@ -511,64 +534,76 @@ void Scope::present(const std::string& indent, int ix)
   }
   else
   {
-    LM_T(LmtPresent, ("%sScope %d:",    
-		      indent.c_str(),
-		      ix));
+    LM_T(LmtPresent, ("%sScope %d:",    indent.c_str(), ix));
   }
 
-  LM_T(LmtPresent, ("%s  Type:     '%s'", 
-		    indent.c_str(), 
-		    type.c_str()));
+  LM_T(LmtPresent, ("%s  Type:     '%s'", indent.c_str(), type.c_str()));
   
   if (oper != "")
-    LM_T(LmtPresent, ("%s  Operator: '%s'", 
-		      indent.c_str(), 
-		      oper.c_str()));
+    LM_T(LmtPresent, ("%s  Operator: '%s'", indent.c_str(), oper.c_str()));
 
   if (areaType == orion::NoArea)
   {
-    LM_T(LmtPresent, ("%s  Value:    %s", 
-		      indent.c_str(), 
-		      value.c_str()));
+    LM_T(LmtPresent, ("%s  Value:    %s", indent.c_str(), value.c_str()));
   }
   else if (areaType == orion::CircleType)
   {
     LM_T(LmtPresent, ("%s  FI-WARE Circle Area:", indent.c_str()));
-    LM_T(LmtPresent, ("%s    Radius:     %s", 
-	  indent.c_str(), 
-	  circle.radiusString().c_str()));
-    LM_T(LmtPresent, ("%s    Longitude:  %s", 
-	  indent.c_str(), 
-	  circle.center.longitudeString().c_str()));
-    LM_T(LmtPresent, ("%s    Latitude:   %s", 
-	  indent.c_str(), 
-	  circle.center.latitudeString().c_str()));
-    LM_T(LmtPresent, ("%s    Inverted:   %s", 
-	  indent.c_str(), 
-	  circle.invertedString().c_str()));
+    LM_T(LmtPresent, ("%s    Radius:     %s",     indent.c_str(), circle.radiusString().c_str()));
+    LM_T(LmtPresent, ("%s    Longitude:  %s",     indent.c_str(), circle.center.longitudeString().c_str()));
```

Comment:
Not sure what you mean here ...
It is indented:

    else if (areaType == orion::CircleType)
    {
      LM_T(LmtPresent, ("%s  FI-WARE Circle Area:", indent.c_str()));
      LM_T(LmtPresent, ("%s    Radius:     %s",     indent.c_str(), circle.radiusString().c_str()));
      LM_T(LmtPresent, ("%s    Longitude:  %s",     indent.c_str(), circle.center.longitudeString().c_str()));
      LM_T(LmtPresent, ("%s    Latitude:   %s",     indent.c_str(), circle.center.latitudeString().c_str()));
      LM_T(LmtPresent, ("%s    Inverted:   %s",     indent.c_str(), circle.invertedString().c_str()));
    }

Radius/Longitude etc all have two spaces more than 'FI-WARE Circle Area'.